### PR TITLE
Fix links in checklist

### PR DESCRIPTION
**Pull request by vernalkick**
cc @gmalette

### CODE REVIEW

**Comment by vernalkick** [comment]
Noticed we're missing `for=` attrs on the labels

File: app/views/deploys/_checklist.html.erb [hunk 1]
```diff
@@ -6,8 +6,8 @@
   <ul class="deploy-checklist">
     <%- stack.checklist.each do |check| -%>
       <li class="deploy-checklist__item">
+        <%= check_box_tag :checklist, nil, false, class: 'deploy-checklist__item__checkbox' %>
         <label class="deploy-checklist__item__label">
-          <%= check_box_tag :checklist, nil, false, class: 'deploy-checklist__item__checkbox' %>
```
Comment (byroot) [inline app/views/deploys/_checklist.html.erb]: Correct me if I'm wrong, but this change won't allow you to click on the text to check the box anymore right?

Should we add an `id` and a `for` attribute?

Comment (gmalette) [inline app/views/deploys/_checklist.html.erb]: yeah he asked me to take care of it. @vernalkick can you add the proper classes such that the "Create Deploy" button will be disabled when it has a `disabled` class?

**Comment by gmalette** [comment]
@vernalkick I added the `id` and `for`

**Comment by vernalkick** [comment]
:cool: :ship: ?

**Comment by gmalette** [comment]
@vernalkick can you add the proper CSS such that the "Create Deploy" button will be disabled when it has a `disabled` class?

**Comment by gmalette** [comment]
then yes, totally :ship:

**Comment by vernalkick** [comment]
Unrelated to this PR?

**Comment by gmalette** [comment]
/shrug depending how you see it. Feel free to merge it if you think it's unrelated

**Comment by byroot** [comment]
@gmalette I think https://github.com/Shopify/shipit-engine/pull/418 fixes this as well.

**Comment by gmalette** [comment]
Ah that makes sense

**Pull request merged**